app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -323,7 +325,7 @@ class ApiServiceTest {
 
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.updateUser(userId, body)
+        val actualResponse = apiService.updateUser(bearerToken, userId, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver